### DISCUSSION TITLE
Publications with ':' cause 'failed to render YAML'

### DISCUSSION THREAD

**Discussion by AnthonyMcDougal** [aidenlx/obsidian-zotero#214] [Help] [answered]
I was getting 'Failed to render YAML' when attempting to create a note for one of my articles. After some experimentation, I found that:

- any article with a publication containing a colon ':' would give the same error
- removing `publication: <%= it.publicationTitle %>` from my zt-field.eta avoided the error

**Comment by aidenlx** [comment, marked as answer]
adding quote before and after field should fix this: 
```yaml
publication: "<%= it.publicationTitle %>"
```

check here for details: https://zotlit.aidenlx.top/getting-started/basic-usage/template-config#note-properties-template
